README-developers.md

```diff
@@ -1,24 +1,42 @@
 # Eel Developers
 
-## Setting up a development environment
+## Setting up your environment
 
 In order to start developing with Eel you'll need to checkout the code, set up a development and testing environment, and check that everything is in order.
 
-Clone the repository:
+### Clone the repository:
 ```bash
-git clone git@github.com:samuelhwilliams/Eel.git
+git clone git@github.com:ChrisKnott/Eel.git
```

Comment:
We probably need to update this now that it's under the `python-eel` org.

Reply:
Nice! I didn't even notice that happened. I'll make that update.

Reply:
Fixed, thanks!